.github/workflows/peek_icons.yml

```
@@ -20,6 +23,18 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r ./.github/scripts/requirements.txt
 
+      - name: Save the pr num in an artifact
+        shell: bash
+        env:
+          PR_NUM: ${{ github.event.number }}
+        run: echo $PR_NUM > pr_num.txt
+
+      - name: Upload the pr num
+        uses: actions/upload-artifact@v2
```

I think it's better to use the `master` version here. It's more updated than the `v2` branch.
```suggestion
        uses: actions/upload-artifact@master
```

---

I think it might be better to stay with v2 or at least a specific tag. That way, if the upload-artifact repo is updated and breaking change occurs, we don't have to worry about it until we upgrade the action.

---

@Thomas-Boi is right, (almost) always stick to a specific version and do not target a `master` or `latest` branch. Targeting a `latest`-release is not secure in terms of code injection attacks. You might wanna take a look at https://github.com/devicons/devicon/issues/395.